src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -1943,9 +1945,15 @@ namespace OutputReportTabular {
 					displayEconomicResultSummary = true;
 					WriteTabularFiles = true;
 					nameFound = true;
-				} else if ( SameString( AlphArray( iReport ), "EnergyMeters" ) ) {
+				} else if (SameString(AlphArray(iReport), "EnergyMeters")) {
 					WriteTabularFiles = true;
 					nameFound = true;
+				} else if (SameString(AlphArray(iReport), "EIO")) {
```

Comment:
No, it is a undocumented shortcut. There are a few others and this one made sense to include to me, primarily for developers.